src/data/exercises.json

```diff
@@ -0,0 +1,455 @@
+{
+    "total_count": 10,
```

Comment:
Correct, this is the result set with 10 exercises.

For an example of what you will get for searching for 1 embed tag, see here: https://exercises-dev1.openstax.org/api/exercises?q=tag:k12phys-ch04-ex001

Searching for multiple tags at once:
https://exercises-dev1.openstax.org/api/exercises?q=tag:k12phys-ch04-ex001,k12phys-ch04-ex002

And if you search for a tag that's in a bunch of exercises, like a LO, you will get a lot of results: https://exercises-dev1.openstax.org/api/exercises?q=tag:k12phys-ch04-s01-lo01